packages/components/src/local/organisms/setting-channels/index.tsx

```diff
@@ -166,14 +178,21 @@ export const SettingChannels: React.FC<PropTypes> = ({
   return (
     <AdminContent>
       <LeftSide>
+        <Dialog
+          isOpen={deleteSelectedChannel}
+          header='Error'
+          content="You can't delete selecting channel"
```

Comment:
`selected`

But, does this prevent users deleting the one last channel?

@lebaphi - What's the reason for not being able to delete the selected channel?

Reply:
Hi @IanMayo ,
> But, does this prevent users deleting the one last channel?

yep it will prevent users deleting the last channel, I will update the fix

> What's the reason for not being able to delete the selected channel?

I found the problem is that when we delete the selected channel, the search list is updated with the new channels but the data in the right panel is not, it still shows the deleted channel. We can fix this but it requires more work to debug/refactor the code. I saw some apps with similar behavior (preventing user from deleting selected item) so I decided to use this solution 🤣

Reply:
Hi @IanMayo, so what approach do you want to take?